src/main/resources/Ideas/IdeasClass.xml

```diff
@@ -515,7 +551,7 @@
         var againstText = $('.result-against').next();
         // Changing the class acording to the vote. Updating the text that shows to the user which one is his choice.
         if(response.result) {
-          voted.forEach( e =&gt; {
+          $.each(voted, function(index, e) {
```

Comment:
You could have kept ``forEach`` but passing a function instead of a lambda expression, as in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach .